Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -222,6 +222,8 @@ static bool probe_calibration_points(float z_pt[NPP + 1], const int8_t probe_poi
 
   if (!_0p_calibration) {
 
+    const float dcr = delta_calibration_radius();
```

Comment:
dcr can be delta_calibration_radius or 0.9*delta_calibration_radius depending on probe grid